fv3core/stencils/dyn_core.py

```diff
@@ -238,7 +238,7 @@ def __init__(
                 self.grid.domain_shape_full(add=(1, 1, 1)), self.grid.full_origin()
             )
 
-            dp_ref_stencil = StencilWrapper(
+            dp_ref_stencil = FrozenStencil(
```

Comment:
I kind of like making FrozenStencil explicitly instead of relying on gtstencil, but I'm interested in hearing what reviewers think. I'm happy to switch it as well.